engine.py

```diff
@@ -254,7 +254,7 @@ def init_engine(self):
         if maya_ver.startswith("Maya "):
             maya_ver = maya_ver[5:]
         if maya_ver.startswith(("2014", "2015", "2016", "2017")):
-            self.log_debug("Running Maya version %s" % maya_ver)
+            self.logger.debug("Running Maya version %s" % maya_ver)
```

Comment:
FYI, this could also be written as
`self.logger.debug("Running Maya version %s", maya_ver)`
This syntax is a bit more robust. If you provide too many or not enough arguments to the `%` operator, an exception will be thrown right here and execution would stop. If you pass them one by one as arguments, the logger will actually log an exception to let you know which logged message failed but execution will resume.

Plus, a log is formatted only if the log level it is being written to is enabled, so you save a few cycles by letting the logger do the formatting.